### DISCUSSION TITLE
Widening and Narrowing

### DISCUSSION THREAD

**Discussion by leehahoon** [prosyslab-classroom/cs524-program-analysis#433] [Art competition]
## Song Title
꿈을 쫓는 소년 (Boy Chasing Dreams)

## Song
https://suno.com/song/ce8b15a9-4528-4d67-9b8e-4372886a8899

## Album Cover
![Boy Chasing Dreams_Widening_and_Narrowing](https://github.com/prosyslab-classroom/cs524-program-analysis/assets/15906121/73765932-9ccc-4374-ac74-3265f5eb2f51)

### [KOR]
**Verse 1:**
처음에는 아무것도 보이지 않았지만
Widening을 통해 길이 보였어
꿈꾸던 세상이 내 앞에 펼쳐지고
희망의 빛이 내 마음을 밝혔어

**Verse 2:**
하지만 점차 가는 길이 너무 많아져서
Narrowing을 통해 길을 줄였어
수많은 선택 속에서 헤매이다가
내가 가야 할 길을 찾았어

**Chorus:**
Widening, 넓어진 세상 속에서
길을 찾았지, 희망의 빛을 봤어
Narrowing, 좁아진 길을 따라가
마침내 난 나의 길을 찾았어

**Outro:**
Widening, narrowing, 나의 길
빛나는 내 꿈을 향해
걸어가, 끝없는 여정 속에
나는 나의 길을 찾아

### [ENG]
**Verse 1:** 
At first, nothing was visible 
Through widening, I saw the path 
The world I dreamed of unfolded before me 
The light of hope brightened my heart

**Verse 2:** 
But gradually, the paths became too many 
Through narrowing, I reduced the paths 
Wandering through countless choices 
I found the path I must take

**Chorus:** 
Widening, in the expanded world 
I found the path, saw the light of hope 
Narrowing, following the narrowed path 
Finally, I found my way

**Outro:** 
Widening, narrowing, my path 
Towards my shining dream 
Walking, in the endless journey 
I found my way

## Name
Jaehoon Jang

## Tool You Used
- [Suno](https://suno.com/): Make a song 
- [ChatGPT](https://chatgpt.com/): Write a lyrics
- [Bing Image Creator](https://www.bing.com/create): Make an album cover image

## Short Description
For me, [*Widening and Narrowing*] means [*A new path in a place where nothing can be seen, A path gradually narrowing down from too many paths*]".

## Long Description (up to 3 sentences)
When starting, he began from the very bottom, in a place where nothing was visible, and sought a new path. 
As he searched, countless paths emerged, so he had to narrow them down and move forward toward his dream. 
After wandering and walking through many paths, the boy finally found his dream.

**Comment by 33577** [comment]
Wow...

**Comment by KihongHeo** [comment]
Very nice.
